sagemaker-pipelines/tabular/tuning-step/tuning_step_example.ipynb

```diff
@@ -0,0 +1,826 @@
+{
```

Comment:
<p>We avoid using pip installs for packages that are installed by default in SageMaker environments. You can remove the sys import and pip install.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>